src/app/technical-record-search/technical-record.service.ts

```diff
@@ -14,17 +14,16 @@ export class TechnicalRecordService {
   constructor(private httpClient: HttpClient,
     private _appConfig: AppConfig) {
     this.routes = {
-      // techRecords: (searchIdentifier: string) => `${this._apiServer.APITechnicalRecordServerUri}/vehicles/${searchIdentifier}/tech-records`,
-      techRecordsAllStatuses: (searchIdentifier: string) => `${this._apiServer.APITechnicalRecordServerUri}/vehicles/${searchIdentifier}/tech-records?status=all&metadata=true`,
+      techRecordsAllStatuses: (searchIdentifier: string, searchCriteria: string) => `${this._apiServer.APITechnicalRecordServerUri}/vehicles/${searchIdentifier}/tech-records?status=all&metadata=true&searchCriteria=${searchCriteria}`,
```

Comment:
break into to two lines.

Reply:
The default value for searchCriteria is 'all'. You don't need to pass this parameter if you are always searching for all.